setup_builder.py

```diff
@@ -136,9 +142,14 @@ def packages(self) -> List[str]:
         """Get a list of directories for Python packages."""
         return self.package_dirs.strip().split()
 
-    def keywords_list(self) -> List[str]:
-        """Get the user-defined keywords as a list."""
-        return self.keywords_spaced.strip().split()
+    def keywords_list(self, base_keywords: List[str]) -> List[str]:
+        """Get the user-defined keywords as a list, along with any base keywords."""
+        keywords = self.keywords_spaced.strip().split() + base_keywords
```

Comment:
There is careful handling (`strip().split()`) here with `self.keywords_spaced` but we append whatever we are handed in `base_keywords`. Are there any checks or handling that would be good to do on `base_keywords` when building up the result?

Reply:
`base_keywords` goes through argparse, so it's already list-ified